examples/ensemble/plot_random_forest_regression_multioutput.py

```diff
@@ -0,0 +1,71 @@
+"""
+============================================================
+Comparing random forests and the multi-output meta estimator
+============================================================
+
+An example to compare multi-output regression with random forest and
+the :ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator.
+
+This example illustrates the use of the
+:ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator
+to perform multi-output regression. A random forest regressor is used,
+which supports multi-output regression natively, so the results can be
+compared.
```

Comment:
<img width="612" alt="download 18" src="https://cloud.githubusercontent.com/assets/1448859/13373117/89f6fe8a-dd5e-11e5-90bc-1b5147e3cfce.png">

Plot shows RFs with different `max_depth`.

I think the hanging in the middle is because the regressor will only ever predict values closer to zero never outside the range of the truth (or truth+noise), so that biases it towards the inside of the circle.

I will add a sentence to that effect.